src/openvic-simulation/map/Region.cpp

```diff
@@ -65,7 +66,7 @@ ProvinceSet::provinces_t const& ProvinceSet::get_provinces() const {
 	return provinces;
 }
 
-static constexpr colour_t ERROR_REGION_COLOUR = COLOUR_COMPONENT << 16;
+static constexpr colour_t ERROR_REGION_COLOUR = colour_t::component << 16;
```

Comment:
We don't want a hardcoded bitshift here to make it red, best solution IMO would be to have static functions for red, green, blue and alpha that take in a component type variable and return a colour with just that component shifted to the right place.